test/tests/build/build_cmake_targets.vader

```diff
@@ -214,14 +202,8 @@ Execute ([CMake targets] Only default build types have default values):
     AssertEqual g:cmake_variants[ 'custom-build' ][ 'cmake_usr_args' ], {'CUSTOM_OP':'ON'}
 
 Execute ([CMake targets] Get default build types):
-    " enable tests for windows with last cmake
-    if !has('win32') || utils#cmake#verNewerOrEq([3, 14])
-        Assert !isdirectory("cmake-build-Release"), "Build directory should not exist"
-        silent CMake
-        Assert filereadable("cmake-build-Release/CMakeCache.txt"), "CMakeCache.txt should be generated"
-        let result = split(cmake4vim#CompleteBuildType(0,0,0), "\n")
-        AssertEqual len(result), 4
-    endif
+    let result = split(cmake4vim#CompleteBuildType(0,0,0), "\n")
```

Comment:
This change shows that something wrong happened. Because before and after each test we remove build folder. It shouldn't work in case if we don't have cached values.

Reply:
This returns the default build types even without a build folder, because `cmake` defines them: `[ 'Default', 'Release', 'RelWithDebInfo, 'MinSizeRel' ]`

I removed the `if` because this should work on windows too, and there is no need for running `CMake`.

Reply:
Yes, right. I thought that it is build targets. Sorry.